Yoast/wordpress-seo
# RW/amp conversion in editor

Fixes #4296 

Testing:
1. Create new post
2. Set ‘Test&’ in the title, heading and first paragraph.
3. Set ‘Test&’ as focus keyword.
4. Check if the focus keyword is detected in the title, heading and paragraph.

Do a similar test for with a category.
